src/test/java/seedu/address/model/contact/ContactTest.java

```diff
@@ -78,8 +79,12 @@ public void equals() {
         editedAlice = new ContactBuilder(ALICE).withEmail(VALID_EMAIL_BOB).build();
         assertFalse(ALICE.equals(editedAlice));
 
+        // different telegram -> returns false
+        editedAlice = new ContactBuilder(ALICE).withTelegram(VALID_TELEGRAM_AMY).build();
+        assertFalse(ALICE.equals(editedAlice));
+
         // different tags -> returns false
-        editedAlice = new ContactBuilder(ALICE).withTags(VALID_TAG_HUSBAND).build();
+        // editedAlice = new ContactBuilder(ALICE).withTags(VALID_TAG_HUSBAND).build();
```

Comment:
Since tags has been implemented for contacts, maybe we should look back at this and update it to also check for tags.

Reply:
Yes, it has been updated. Thanks for the note!